### PR TITLE
655 Fix to return correct column type for spatial datatypes

### PR DESCRIPTION
**Pull request by peterbae**
Fixes issue #655. Also added test cases for it.

Also removed an unused import from Geometry/Geography classes.

### CODE REVIEW

**Comment by peterbae** [comment]
this appveyor failure is unrelated to this PR, it's a bug I found during internal testing that occurs only with SQL Server 2008. I will merge this PR after that issue has been solved.

**Comment by peterbae** [comment]
Actually I'm going to use a workaround for this problem, it seems like it could take a while to solve.

File: src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResultSetMetaData.java [hunk 1]
```diff
@@ -125,6 +125,14 @@ public int getColumnType(int column) throws SQLServerException {
         if ( SSType.SQL_VARIANT == typeInfo.getSSType()){
             jdbcType = JDBCType.SQL_VARIANT;
         }
+        if (SSType.UDT == typeInfo.getSSType()) {
+            if (typeInfo.getSSTypeName().equalsIgnoreCase("geometry")) {
```
Comment (rene-ye) [inline src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResultSetMetaData.java]: I don't really have a problem with this, but why are we doing type check in 3 different ways. VARIANT is being checked separately outside the switch, and now GEOMETRY/GEOGRAPHY will be checked by name. Can we maybe stick to one form? This Microsoft [overview](https://docs.microsoft.com/en-us/sql/relational-databases/spatial/spatial-data-types-overview) of spatial datatypes seems to imply that it's only available in SQL Server 2012 and later, so shouldn't this be in the isKatamaiOrLater() block?

Comment (peterbae) [reply src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResultSetMetaData.java]: We can't stick to one form. If you debug this part of the code, you'll see that the getSSType will give SQL_VARIANT or UDT. If it's UDT, we need to look at the SSTypeName to find out if it's Geometry or Geography, so this is the only way to check for these datatypes.

The overview page mentions that new spatial features have been introduced in SQL Server 2012, but this does not mean that spatial datatypes are supported starting from 2012 - it's supported starting from 2008. so we need this check for 2008 and up.

Comment (cheenamalhotra) [reply src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResultSetMetaData.java]: Please create Constants for these strings and reference them here.

**Pull request merged**